modules/ROOT/pages/tutorial-initial-setup.adoc

```diff
@@ -0,0 +1,101 @@
+= Prerequisites
+
+Those tutorials are written targeting a Linux environment with a working `libvirt` setup and hardware virtualisation support via `KVM`. For instructions to setup `libvirt` and `KVM` you may refer to the link:https://docs.fedoraproject.org/en-US/quick-docs/getting-started-with-virtualization/[Getting started with virtualization] guide from Fedora. Although this setup guide is focused on Fedora, those tutorials should work on any distribution with `libvirt` installed.
+
+== Downloading a Fedora CoreOS QCOW2 image
+
+First of all, we have to download the latest stable release of Fedora CoreOS. As we will use `libvirt`, we will download an archive containing a QEMU image from link:https://getfedora.org/en/coreos/download?tab=metal_virtualized&stream=stable[getfedora.org]:
+
+[source,bash]
+----
+RELEASE="32.20200715.3.0" # Update this with the latest release from https://getfedora.org/en/coreos
+curl -O https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/$RELEASE/x86_64/fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz
+curl -O https://builds.coreos.fedoraproject.org/prod/streams/stable/builds/$RELEASE/x86_64/fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz.sig
+----
+
+Once the archive has been downloaded, make sure to verify its integrity by following the instructions available by clicking on the `Verify signature & SHA256` button. You will have to download the checksum file, the signature and Fedora GPG keys to verify your download:
+
+[source,bash]
+----
+curl https://getfedora.org/static/fedora.gpg | gpg --import
+gpg --verify fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz.sig fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz
+----
+
+Once you have verified the archive, you can extract it with:
+
+[source,bash]
+----
+unxz fedora-coreos-$RELEASE-qemu.x86_64.qcow2.xz
+----
```

Comment:
That's a good idea. But it looks like we don't ship coreos-installer binaries so that would mean extracting them from the official container or running a long podman command to but the qcow2 in the right place.

Reply:
> running a long podman command to put the qcow2 in the right place

IMHO this is indeed what we should be teaching people to do. It maybe be slightly more annoying to document, but it's a long term win.

Reply:
We have rpms for fedora OR container for non-fedora. I think either one of those should suffice. We could put more detail into a "how to download" and one section of that page would be options for how to get/use `coreos-installer`.